util/util.go

```diff
@@ -148,3 +148,11 @@ func RemoveOrphanedInstances(verbose bool) error {
 
 	return nil
 }
+
+// AbsTarPathMatches tells whether the tar header name matches the path.
+// This function is needed since some tar files prefix its header names
+// with / and some not. NOTE: argument 'path' should start with '/' while
+// 'tarPath' should be header.Name.
+func AbsTarPathMatches(path, tarPath string) bool {
+	return tarPath == path || "/"+tarPath == path
```

Comment:
But I like your regex solution, especially since it enables me to use regex as `path` argument. This way I can simplify [this line](https://github.com/mikelangelo-project/capstan/blob/master/cmd/package.go#L429) and get rid of the ugly comment there. So thanks for the goplay 👍